synapse/lib/config.py

```diff
@@ -392,28 +392,20 @@ def reqConfValid(self):
         else:
             return
 
-    def reqConfValu(self, key):
-        '''
-        Get a configuration value.  If that value is not present in the schema
-        or is not set, then raise an exception.
-
-        Args:
-            key (str): The key to require.
+    def reqConfValu(self, name):
+        s_common.deprecated('reqConfValu()')
```

Comment:
This isn't a deprecation; but a new API under the old name. We should discuss this.

Reply:
The old API name is what's deprecated. We've definitely done this before.